README.md

```diff
@@ -0,0 +1 @@
+Quote Approval Bot
```

Comment:
Could you please add the following to the README:

1. Brief description of the app
2. A list of required / optional `env` vars
3. Link to the Slack app
4. Any specifics around building / deploying